main.cpp

```diff
@@ -42,9 +44,17 @@ double log_likelihood (const SampleType &prm)
                                           3.19, 1.73, 1.94, 2.14, 2.91, 2.85,
                                           2.8, 2.37, 2.42, 2.68, 2.01, 1.9,
                                           2.14};
+*/
+  const Data::PomData pom_data;
+  const std::valarray<double> dataDiam = pom_data.tem_diam_time2;
 
+  std::vector<std::valarray<double>> fitData(1);
```

Comment:
```suggestion
  std::vector<std::valarray<double>> fitData = { std::pow(dataDiam/0.3000805, 3) };
```
This way you initialize right away and can, if you want, make the object `const`.